scripts/compare-features.js

```diff
@@ -4,15 +4,16 @@
 'use strict';
 
 /**
- *
  * Sort a list of features based upon a specific order:
  *  1. __compat is always first
  *  2. Alphanumerical features starting with an uppercase letter (without symbols aside from - or _)
  *  3. Alphanumerical features starting with a lowercase letter (without symbols aside from - or _)
  *  4. All other features
  *
+ * @param {string} a The name of the first object to perform comparison with
+ * @param {string} b The name of the second object to perform comparison with
```

Comment:
To make the parameter names be more visually distinct from the description, it would actually be good to add ` - ` between them (everywhere).

```suggestion
 * @param {string} a - The name of the first object to perform comparison with
 * @param {string} b - The name of the second object to perform comparison with
```